### DISCUSSION TITLE
Cómo mostrar objetos por medio de parámetros hacia twig

### DISCUSSION THREAD

**Discussion by AlanJair** [pimcore/pimcore#14710] [Q&A / Support]
He intentado pasar objetos que tengo en la carpeta `Proyectos` por medio de parámetros de esta manera:
```php
public function projectsAction(Request $request)
    {
        $paramsPro = [];
        $projectObjectsFolder = Service::createFolderByPath('/Proyectos/');
        $paramsPro['projects'] = projectObjectsFolder->getChildren();

        return $this->render('content/editProjects.html.twig', $paramsPro);
    }
``` 
y en `content/editProjects.html.twig` tengo:
```twig
{%  for project in projects %}
<div class="col-md-6 col-lg-3 d-flex align-items-stretch" >
    <h4 class="title">{{ project.projectName }}</h4>
    <h4 class="description">{{ project.description }}</h4>
</div>
 {% endfor %}
``` 
Pero no me funciona y en `'content/editProjects.html.twig'` me dice "Variable "projects" does not exist."

¿Cómo puedo pasar por parámetros los DataObjects de la carpeta `Proyectos` a una plantilla Twig o habrá alguna otra forma de mostrar los DataObjects a una plantilla Twig en específico o como esta manera?

**Comment by podarcis** [comment]
> Cómo mostrar objetos por medio de parámetros hacia twig

Posting in English instead of Spanish will significantly improve your chances of getting a proper answer to your question.

**Reply by mattamon** [reply]
> ¿Cómo puedo pasar por parámetros los DataObjects de la carpeta `Proyectos` a una plantilla Twig o habrá alguna otra forma de mostrar los DataObjects a una plantilla Twig en específico o como esta manera?

Can you try to just dump everything in the twig file?
You can do this by calling `   {{ dump() }}` anywhere in the twig file.
Check your params there. 

In general for getting specific objects I would recommend using Listings instead of getting all children from a folder.
https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Working_with_PHP_API.html#page_Object-Listings -> there you can add conditions to check if the objects are in the projects folder.

**Reply by AlanJair** [reply]
También lo intente de esa manera, gracias.

Al parecer todo el tiempo fue problema del servidor, me di cuenta que en la demo de pimcore tampoco me funcionaban ciertas conversiones, así que decidí reiniciar mi servidor Apache y de esta manera funciono correctamente hasta cierto tiempo. Aunque nose la razón de esto, talvez sea el rendimiento de mi equipo.

**Reply by mattamon** [reply]
Ah okay.
But the controllers basically work like in Symfony so there is not a big difference here.
Is your problem resolved then?

**Reply by AlanJair** [reply]
Si, gracias!
